lib/spack/spack/util/web.py

```diff
@@ -243,13 +264,63 @@ def _spider_wrapper(args):
     return _spider(*args)
 
 
-def _urlopen(*args, **kwargs):
+# TODO(opadron): There's gotta be a better place for stuff like this.
```

Comment:
@tgamblin @scheibelp @alalazo @adamjstewart @scottwittenburg Thoughts?

(Also, is there a way to mention reviewers, like `@reviewers`?  Would be nice.)

Reply:
I don't understand what the purpose of `noopify` is or why we need it.  Can you explain what you're really trying to do here?

Reply:
I was writing an explanation, but in doing so I decided that it was just way too complicated for what we need.  I think I should just get rid of it.